<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="A.DeMartin" />

<meta name="date" content="2024-12-16" />

<title>ColonicFb</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ColonicFb</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">ColonicFb</h1>
<h4 class="author">A.DeMartin</h4>
<h4 class="date">2024-12-16</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-03-05
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>ColonicFb/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20241216code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20241216)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20241216code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20241216)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/ColonicFb/data/allmerged_seurat.rds
</td>
<td style="text-align:left;">
data/allmerged_seurat.rds
</td>
</tr>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/markerGenes_res0.25
</td>
<td style="text-align:left;">
analysis/markerGenes_res0.25
</td>
</tr>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/DEGenes_cond
</td>
<td style="text-align:left;">
analysis/DEGenes_cond
</td>
</tr>
<tr>
<td style="text-align:left;">
/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/DEGenesAdamdec1_cond
</td>
<td style="text-align:left;">
analysis/DEGenesAdamdec1_cond
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomangeldemartinColonicFbtree38aa622eff0de8146d657049642a5b5dbe62aa67targetblank38aa622a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/angeldemartin/ColonicFb/tree/38aa622eff0de8146d657049642a5b5dbe62aa67" target="_blank">38aa622</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomangeldemartinColonicFbtree38aa622eff0de8146d657049642a5b5dbe62aa67targetblank38aa622a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/angeldemartin/ColonicFb/tree/38aa622eff0de8146d657049642a5b5dbe62aa67" target="_blank">38aa622</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.Rhistory
    Ignored:    analysis/figure/
    Ignored:    data/colonicFb scdata/

Untracked files:
    Untracked:  analysis/DEGenesAdamdec1_cond

Unstaged changes:
    Modified:   analysis/MarkerGenes.Rmd
    Modified:   analysis/Merge_QC_UMAPS.Rmd
    Modified:   analysis/RelAbundance.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/MarkerGenes.Rmd</code>) and HTML
(<code>docs/MarkerGenes.html</code>) files. If you’ve configured a
remote Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/38aa622eff0de8146d657049642a5b5dbe62aa67/analysis/MarkerGenes.Rmd" target="_blank">38aa622</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-19
</td>
<td>
update gitignore
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/ColonicFb/38aa622eff0de8146d657049642a5b5dbe62aa67/docs/MarkerGenes.html" target="_blank">38aa622</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-19
</td>
<td>
update gitignore
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/analysis/MarkerGenes.Rmd" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
<td>
Dez24
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/angeldemartin/ColonicFb/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/MarkerGenes.html" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
<td>
Dez24
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="load-packages" class="section level2">
<h2>load packages</h2>
</div>
<div id="load-file" class="section level2">
<h2>load file</h2>
<pre class="r"><code>##load merged file 
fileNam &lt;- &quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/data/allmerged_seurat.rds&quot;
seuratM &lt;- readRDS(fileNam)</code></pre>
</div>
<div id="set-color-vectors" class="section level2">
<h2>set color vectors</h2>
<pre class="r"><code>## set vector for cond
colcond &lt;- c(&quot;#202547&quot;,&quot;#99B898&quot;, &quot;#BE3144&quot;)
names(colcond) &lt;- c(&quot;untreated&quot;, &quot;crod&quot;, &quot;DSS&quot;)

## set vector for dataset
coldataset &lt;- c(&quot;#202547&quot;,&quot;#99B898&quot;, &quot;#BE3144&quot;)
names(coldataset) &lt;- c(&quot;372561_14-14_20241203_Mu_colonicFb_untreated&quot;, &quot;372561_15-15_20241203_Mu_colonicFb_cond1&quot;, &quot;372561_16-16_20241209_Mu_colonicFb_cond2&quot;)

colpal &lt;- c(&quot;#355C7D&quot;,&quot;#628395&quot;,&quot;#99B898&quot;,&quot;#66C2A5&quot;,&quot;#E84A5F&quot;,&quot;#B45B5C&quot;,&quot;#BF782D&quot;,&quot;#6C5B7B&quot;, &quot;#C06C84&quot;, &quot;#c7eae5&quot;,&quot;#904D39&quot;, &quot;#67001f&quot;,&quot;#0F1F38&quot;,&quot;#8E9B97&quot;)
names(colpal) &lt;- c(&quot;0&quot;,&quot;3&quot;,&quot;2&quot;, &quot;6&quot;, &quot;1&quot;, &quot;5&quot;, &quot;4&quot;, &quot;7&quot;,&quot;8&quot;, &quot;11&quot;, &quot;9&quot;, &quot;12&quot;, &quot;10&quot;, &quot;13&quot;)

colclusterName &lt;- c(&quot;#355C7D&quot;,&quot;#628395&quot;,&quot;#99B898&quot;,&quot;#66C2A5&quot;,&quot;#E84A5F&quot;,&quot;#B45B5C&quot;,&quot;#BF782D&quot;,&quot;#6C5B7B&quot;, &quot;#C06C84&quot;, &quot;#c7eae5&quot;,&quot;#904D39&quot;, &quot;#67001f&quot;,&quot;#0F1F38&quot;,&quot;#8E9B97&quot;)
names(colclusterName) &lt;- c(&quot;PdgfraloFb1&quot;,&quot;PdgfraloFb2&quot;,&quot;Trophocytes1&quot;,&quot;Trophocytes2&quot;,&quot;Telocytes/Fibro2&quot;,&quot;PdgfrahiFb&quot;,&quot;Myocytes&quot;,&quot;Glial&quot;,&quot;ICC1&quot;, &quot;ICC2&quot;,&quot;Immun1&quot;,&quot;Immun2&quot;,&quot;EpC&quot;,&quot;Cycling&quot;)


#c(&quot;#67001f&quot;, &quot;#D53E4F&quot;, &quot;#f4a582&quot;, &quot;#FEE08B&quot;, &quot;#003c30&quot;,&quot;#01665e&quot;,&quot;#66C2A5&quot;, &quot;#3288BD&quot;,&quot;#BEAED4&quot;, &quot;#c7eae5&quot;,&quot;#355C7D&quot;,&quot;#202547&quot;,&quot;#B45B5C&quot;,&quot;#8c510a&quot;)
#c(&quot;#dfc27d&quot;,&quot;#BE3144&quot;,&quot;#202547&quot;,&quot;#355C7D&quot;,&quot;#779d8d&quot;)
#c(&quot;#202547&quot;,&quot;#BE3144&quot;)</code></pre>
</div>
<div id="calculate-cluster-marker-genes" class="section level2">
<h2>calculate cluster marker genes</h2>
<pre class="r"><code>##cluster marker
Idents(seuratM) &lt;- seuratM$RNA_snn_res.0.25
markerGenes &lt;- FindAllMarkers(seuratM, only.pos=T)  %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01)

#save table
write.table(markerGenes, 
            file= &quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/markerGenes_res0.25&quot;,
            sep=&quot;\t&quot;,
            quote=F,
            row.names=F,
            col.names=T)</code></pre>
</div>
<div id="extract-genes-for-heatmap" class="section level2">
<h2>extract genes for heatmap</h2>
<pre class="r"><code>## load DEGenesFbcond3
markerGenes &lt;- read.delim(&quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/markerGenes_res0.25&quot;, header = TRUE, sep = &quot;\t&quot;)

##adjust table
markerGenes &lt;- markerGenes %&gt;%
  mutate(Gene=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))  %&gt;%
  mutate(EnsID=gsub(&quot;\\..*&quot;,&quot;&quot;, gene))

marker_0 &lt;- dplyr::filter(markerGenes, cluster == &quot;0&quot;) %&gt;%     # Filter rows where cluster is &quot;0&quot;
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% # Select the 10 rows with the smallest p_val_adj
  slice_max(order_by = avg_log2FC, n = 10) # Select the 10 rows with the smallest p_val_adj

marker_1 &lt;- dplyr::filter(markerGenes, cluster == &quot;1&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10) 

marker_2 &lt;- dplyr::filter(markerGenes, cluster == &quot;2&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10) 

marker_3 &lt;- dplyr::filter(markerGenes, cluster == &quot;3&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10) 

marker_4 &lt;- dplyr::filter(markerGenes, cluster == &quot;4&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10) 

marker_5 &lt;- dplyr::filter(markerGenes, cluster == &quot;5&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10) 

marker_6 &lt;- dplyr::filter(markerGenes, cluster == &quot;6&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10) 

marker_7 &lt;- dplyr::filter(markerGenes, cluster == &quot;7&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10) 

marker_8 &lt;- dplyr::filter(markerGenes, cluster == &quot;8&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10)

marker_9 &lt;- dplyr::filter(markerGenes, cluster == &quot;9&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10)

marker_10 &lt;- dplyr::filter(markerGenes, cluster == &quot;10&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10)

marker_11 &lt;- dplyr::filter(markerGenes, cluster == &quot;11&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10)

marker_12 &lt;- dplyr::filter(markerGenes, cluster == &quot;12&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10)

marker_13 &lt;- dplyr::filter(markerGenes, cluster == &quot;13&quot;) %&gt;%  
  slice_min(order_by = p_val_adj, n = 10)  %&gt;% 
  slice_max(order_by = avg_log2FC, n = 10)

# List of data frames
df_list &lt;- list(marker_0, marker_1, marker_2, marker_3, marker_4, marker_5, marker_6, marker_7, marker_8, marker_9, marker_10, marker_11, marker_12, marker_13)
marker_all &lt;- reduce(df_list, full_join)
mGenes &lt;- as.vector(marker_all$Gene)</code></pre>
</div>
<div id="average-heatmap-top10-marker" class="section level2">
<h2>average heatmap top10 marker</h2>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$clusterName
seurat &lt;- seuratM
levels(seurat)</code></pre>
<pre><code> [1] &quot;PdgfraloFb1&quot;      &quot;PdgfraloFb2&quot;      &quot;Trophocytes1&quot;     &quot;Trophocytes2&quot;     &quot;Telocytes/Fibro2&quot;
 [6] &quot;PdgfrahiFb&quot;       &quot;Myocytes&quot;         &quot;Glial&quot;            &quot;ICC1&quot;             &quot;ICC2&quot;            
[11] &quot;Immun1&quot;           &quot;Immun2&quot;           &quot;EpC&quot;              &quot;Cycling&quot;         </code></pre>
<pre class="r"><code>avgHeatmap &lt;- function(seurat, selGenes, colVecIdent, colVecCond=NULL,
                       ordVec=NULL, gapVecR=NULL, gapVecC=NULL,cc=FALSE,
                       cr=FALSE, condCol=FALSE){
  
  selGenes &lt;- selGenes$gene
  
  ## assay data
  clusterAssigned &lt;- as.data.frame(Idents(seurat)) %&gt;%
  dplyr::mutate(cell=rownames(.))
  colnames(clusterAssigned)[1] &lt;- &quot;ident&quot;
  seuratDat &lt;- GetAssayData(seurat)
  
  ## genes of interest
  genes &lt;- data.frame(gene=rownames(seurat)) %&gt;% 
    mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene)) %&gt;% filter(geneID %in% selGenes)

  ## matrix with averaged cnts per ident
  logNormExpres &lt;- as.data.frame(t(as.matrix(
    seuratDat[which(rownames(seuratDat) %in% genes$gene),])))
  logNormExpres &lt;- logNormExpres %&gt;% dplyr::mutate(cell=rownames(.)) %&gt;%
    dplyr::left_join(.,clusterAssigned, by=c(&quot;cell&quot;)) %&gt;%
    dplyr::select(-cell) %&gt;% dplyr::group_by(ident) %&gt;%
    dplyr::summarise_all(mean)
  logNormExpresMa &lt;- logNormExpres %&gt;% dplyr::select(-ident) %&gt;% as.matrix()
  rownames(logNormExpresMa) &lt;- logNormExpres$ident
  logNormExpresMa &lt;- t(logNormExpresMa)
  rownames(logNormExpresMa) &lt;- gsub(&quot;^.*?\\.&quot;,&quot;&quot;,rownames(logNormExpresMa))
  
  ## remove genes if they are all the same in all groups
  ind &lt;- apply(logNormExpresMa, 1, sd) == 0
  logNormExpresMa &lt;- logNormExpresMa[!ind,]
  genes &lt;- genes[!ind,]

  ## color columns according to cluster
  annotation_col &lt;- as.data.frame(colnames(logNormExpresMa))
    #dplyr::mutate(celltype=colnames(logNormExpresMa))
  #colnames(annotation_col)[1] &lt;- &quot;cond&quot;
  colnames(annotation_col)[1] &lt;- &quot;celltype&quot;
    annotation_col &lt;- annotation_col %&gt;%
    dplyr::mutate(celltype = gsub(&quot;(^.*?_)&quot;,&quot;&quot;,celltype))
 rownames(annotation_col) &lt;- colnames(logNormExpresMa) 

  ann_colors = list(
      cond = colcond,
      celltype=colVecIdent)
  if(is.null(ann_colors$cond)){
    annotation_col$cond &lt;- NULL
  }
  
  ## adjust order
  logNormExpresMa &lt;- logNormExpresMa[selGenes,]
  if(is.null(ordVec)){
    ordVec &lt;- levels(seurat)
    ordVec &lt;- c(&quot;PdgfraloFb1&quot;,&quot;PdgfraloFb2&quot;,&quot;Trophocytes1&quot;,&quot;Trophocytes2&quot;,&quot;Telocytes/Fibro2&quot;,&quot;PdgfrahiFb&quot;,&quot;Myocytes&quot;,&quot;Glial&quot;,&quot;ICC1&quot;, &quot;ICC2&quot;,&quot;Immun1&quot;,&quot;Immun2&quot;,&quot;EpC&quot;,&quot;Cycling&quot;)
     
  }
  logNormExpresMa &lt;- logNormExpresMa[,ordVec]

  ## scaled row-wise
  pheatmap(logNormExpresMa, scale=&quot;row&quot; ,treeheight_row = 0, cluster_rows = cr, 
         cluster_cols = cc,
         color = colorRampPalette(c(&quot;#2166AC&quot;, &quot;#F7F7F7&quot;, &quot;#B2182B&quot;))(50),
         annotation_col = annotation_col, cellwidth=15, cellheight=10,
         annotation_colors = ann_colors, gaps_row = gapVecR, gaps_col = gapVecC)
}

##genes
genesPlot &lt;- data.frame(gene=mGenes)

levels(seurat)</code></pre>
<pre><code> [1] &quot;PdgfraloFb1&quot;      &quot;PdgfraloFb2&quot;      &quot;Trophocytes1&quot;     &quot;Trophocytes2&quot;     &quot;Telocytes/Fibro2&quot;
 [6] &quot;PdgfrahiFb&quot;       &quot;Myocytes&quot;         &quot;Glial&quot;            &quot;ICC1&quot;             &quot;ICC2&quot;            
[11] &quot;Immun1&quot;           &quot;Immun2&quot;           &quot;EpC&quot;              &quot;Cycling&quot;         </code></pre>
<pre class="r"><code>colVec &lt;- colclusterName
# colVec &lt;- c(colPal, colPal, colPal)
avgHeatmap(seurat, selGenes = genesPlot, colVecIdent = colVec)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/avg%20heatmap%20top10%20cluster%20marker-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-avg-heatmap-top10-cluster-marker-1">
Past versions of "avg heatmap top10 cluster marker-1.png"
</button>
</p>
<div id="fig-no-spaces-avg-heatmap-top10-cluster-marker-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/avg heatmap top10 cluster marker-1.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dotplot-marker-cluster" class="section level2">
<h2>dotplot marker cluster</h2>
<pre class="r"><code>genes &lt;- data.frame(gene=rownames(seuratM)) %&gt;% 
  mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))

selGenes &lt;- data.frame(geneID=c(&quot;Mki67&quot;,&quot;Epcam&quot;,&quot;Ptprc&quot;,&quot;Ano1&quot;,&quot;Plp1&quot;,&quot;Acta2&quot;,&quot;Bmp5&quot;,&quot;Adamdec1&quot;,&quot;Pdgfra&quot;,&quot;Pi16&quot;,&quot;Cd81&quot;,&quot;Cd34&quot;,&quot;Pdpn&quot;)) %&gt;% left_join(., genes, by=&quot;geneID&quot;)

DotPlot(seuratM, features = selGenes, group.by= &quot;clusterName&quot;) + RotatedAxis() + scale_color_viridis(option=&quot;F&quot;) + coord_flip()</code></pre>
<p><img src="figure/MarkerGenes.Rmd/dotplot%20marker%20cluster-1.png" width="864" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-dotplot-marker-cluster-1">
Past versions of "dotplot marker cluster-1.png"
</button>
</p>
<div id="fig-no-spaces-dotplot-marker-cluster-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/38aa622eff0de8146d657049642a5b5dbe62aa67/docs/figure/MarkerGenes.Rmd/dotplot marker cluster-1.png" target="_blank">38aa622</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dotplot-marker-cluster-fb-only" class="section level2">
<h2>dotplot marker cluster fb only</h2>
<pre class="r"><code>###subset fb 
seuratFb &lt;- subset(seuratM, clusterName %in% c(&quot;PdgfraloFb1&quot;,&quot;PdgfraloFb2&quot;,&quot;Trophocytes1&quot;,&quot;Trophocytes2&quot;,&quot;Telocytes/Fibro2&quot;,&quot;PdgfrahiFb&quot;,&quot;Myocytes&quot;,&quot;Glial&quot;,&quot;ICC1&quot;, &quot;ICC2&quot;))

genes &lt;- data.frame(gene=rownames(seuratFb)) %&gt;% 
  mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))

selGenes &lt;- data.frame(geneID=c(&quot;Ano1&quot;,&quot;Plp1&quot;,&quot;Acta2&quot;,&quot;Bmp5&quot;,&quot;Adamdec1&quot;,&quot;Pdgfra&quot;,&quot;Pi16&quot;,&quot;Cd81&quot;,&quot;Cd34&quot;,&quot;Pdpn&quot;)) %&gt;% left_join(., genes, by=&quot;geneID&quot;)
DotPlot(seuratFb, features = selGenes, group.by= &quot;clusterName&quot;) + RotatedAxis() + scale_color_viridis(option=&quot;F&quot;) + coord_flip()</code></pre>
<p><img src="figure/MarkerGenes.Rmd/dotplot%20marker%20cluster%20fb%20only-1.png" width="864" style="display: block; margin: auto;" /></p>
<pre class="r"><code>selGenes &lt;- data.frame(geneID=c(&quot;Wnt2b&quot;,&quot;Ccn2&quot;,&quot;Bmp4&quot;,&quot;Bmp6&quot;,&quot;Sfrp2&quot;,&quot;Col15a1&quot;,&quot;Grem1&quot;,&quot;Igfbp3&quot;)) %&gt;% left_join(., genes, by=&quot;geneID&quot;)
DotPlot(seuratFb, features = selGenes, group.by= &quot;clusterName&quot;) + RotatedAxis() + scale_color_viridis(option=&quot;F&quot;) + coord_flip()</code></pre>
<p><img src="figure/MarkerGenes.Rmd/dotplot%20marker%20cluster%20fb%20only-2.png" width="864" style="display: block; margin: auto;" /></p>
</div>
<div id="dotplot-marker-clustercond" class="section level2">
<h2>dotplot marker cluster+cond</h2>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$clusterName_cond

genes &lt;- data.frame(gene=rownames(seuratM)) %&gt;% 
  mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))

selGenes &lt;- data.frame(geneID=c(&quot;Mki67&quot;,&quot;Epcam&quot;,&quot;Ptprc&quot;,&quot;Ano1&quot;,&quot;Plp1&quot;,&quot;Acta2&quot;,&quot;Bmp5&quot;,&quot;Adamdec1&quot;,&quot;Pdgfra&quot;,&quot;Pi16&quot;,&quot;Cd81&quot;,&quot;Cd34&quot;,&quot;Pdpn&quot;)) %&gt;% left_join(., genes, by=&quot;geneID&quot;)

DotPlot(seuratM, features = selGenes, group.by= &quot;clusterName_cond&quot;) + RotatedAxis() + scale_color_viridis(option=&quot;F&quot;) + coord_flip()</code></pre>
<p><img src="figure/MarkerGenes.Rmd/dotplot%20marker%20cluster%20plus%20cond-1.png" width="1344" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-dotplot-marker-cluster-plus-cond-1">
Past versions of "dotplot marker cluster plus cond-1.png"
</button>
</p>
<div id="fig-no-spaces-dotplot-marker-cluster-plus-cond-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/38aa622eff0de8146d657049642a5b5dbe62aa67/docs/figure/MarkerGenes.Rmd/dotplot marker cluster plus cond-1.png" target="_blank">38aa622</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="feature-plots-fb-maker-genes" class="section level2">
<h2>feature plots fb maker genes</h2>
<pre class="r"><code>## plot feature
FeaturePlot(seuratM, features = &quot;ENSMUSG00000020717.Pecam1&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-1">
Past versions of "feature plots fb-1.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-1.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000045394.Epcam&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-2">
Past versions of "feature plots fb-2.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-2.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000026395.Ptprc&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-3.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-3">
Past versions of "feature plots fb-3.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-3.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000039542.Ncam1&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-4.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-4">
Past versions of "feature plots fb-4.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-4" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-4.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000031004.Mki67&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-5.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-5">
Past versions of "feature plots fb-5.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-5" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-5.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000028583.Pdpn&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-6.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-6">
Past versions of "feature plots fb-6.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-6" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-6.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000029231.Pdgfra&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-7.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-7">
Past versions of "feature plots fb-7.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-7" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-7.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000035783.Acta2&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-8.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-8">
Past versions of "feature plots fb-8.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-8" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-8.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000037706.Cd81&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-9.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-9">
Past versions of "feature plots fb-9.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-9" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-9.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000016494.Cd34&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-10.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-10">
Past versions of "feature plots fb-10.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-10" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-10.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>FeaturePlot(seuratM, features = &quot;ENSMUSG00000022057.Adamdec1&quot;, cols = c(&quot;lightgrey&quot;, &quot;#BE3144&quot;), pt.size = 1)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/feature%20plots%20fb-11.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-feature-plots-fb-11">
Past versions of "feature plots fb-11.png"
</button>
</p>
<div id="fig-no-spaces-feature-plots-fb-11" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/feature plots fb-11.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="calculate-de-genes-cond" class="section level2">
<h2>calculate DE genes cond</h2>
<pre class="r"><code>##cluster marker
Idents(seuratM) &lt;- seuratM$cond
markerGenes &lt;- FindAllMarkers(seuratM, only.pos=T)  %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01)

#save table
write.table(markerGenes, 
            file= &quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/DEGenes_cond&quot;,
            sep=&quot;\t&quot;,
            quote=F,
            row.names=F,
            col.names=T)</code></pre>
</div>
<div id="gsea-untreated" class="section level2">
<h2>GSEA untreated</h2>
<pre class="r"><code>## load DEGenesFbcond3
DEGenes &lt;- read.delim(&quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/DEGenes_cond&quot;, header = TRUE, sep = &quot;\t&quot;)

##adjust table
DEGenes &lt;- DEGenes %&gt;%
  mutate(Gene=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))  %&gt;%
  mutate(EnsID=gsub(&quot;\\..*&quot;,&quot;&quot;, gene))

DEuntreated &lt;- DEGenes %&gt;% dplyr::filter(cluster == &quot;untreated&quot;)
##GSEA untreated
egountreated&lt;- enrichGO(gene = unique(DEuntreated$EnsID),
                          OrgDb = org.Mm.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egountreated &lt;- setReadable(egountreated, OrgDb = org.Mm.eg.db)
dotplot(egountreated, showCategory=30)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/GSEA%20untreated-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-GSEA-untreated-1">
Past versions of "GSEA untreated-1.png"
</button>
</p>
<div id="fig-no-spaces-GSEA-untreated-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/GSEA untreated-1.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="gsea-c.rod" class="section level2">
<h2>GSEA c.rod</h2>
<pre class="r"><code>DEcrod &lt;- DEGenes %&gt;% dplyr::filter(cluster == &quot;crod&quot;)
##GSEA crod
egocrod&lt;- enrichGO(gene = unique(DEcrod$EnsID),
                          OrgDb = org.Mm.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egocrod &lt;- setReadable(egocrod, OrgDb = org.Mm.eg.db)
dotplot(egocrod, showCategory=30)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/GSEA%20crod-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-GSEA-crod-1">
Past versions of "GSEA crod-1.png"
</button>
</p>
<div id="fig-no-spaces-GSEA-crod-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/GSEA crod-1.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="gsea-dss" class="section level2">
<h2>GSEA DSS</h2>
<pre class="r"><code>DEDSS &lt;- DEGenes %&gt;% dplyr::filter(cluster == &quot;DSS&quot;)
##GSEA DSS
egoDSS&lt;- enrichGO(gene = unique(DEDSS$EnsID),
                          OrgDb = org.Mm.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egoDSS &lt;- setReadable(egoDSS, OrgDb = org.Mm.eg.db)
dotplot(egoDSS, showCategory=30)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/GSEA%20DSS-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-GSEA-DSS-1">
Past versions of "GSEA DSS-1.png"
</button>
</p>
<div id="fig-no-spaces-GSEA-DSS-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/GSEA DSS-1.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cluster-top-100-de-genes" class="section level2">
<h2>cluster top 100 DE genes</h2>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$cond
#top DE genes PdgfraloFb1
seuratPdgfraloFb1 &lt;- subset(seuratM, clusterName == &quot;PdgfraloFb1&quot;)
table(seuratPdgfraloFb1$clusterName)</code></pre>
<pre><code>
     PdgfraloFb1      PdgfraloFb2     Trophocytes1     Trophocytes2 Telocytes/Fibro2 
            7884                0                0                0                0 
      PdgfrahiFb         Myocytes            Glial             ICC1             ICC2 
               0                0                0                0                0 
          Immun1           Immun2              EpC          Cycling 
               0                0                0                0 </code></pre>
<pre class="r"><code>levels(seuratPdgfraloFb1)</code></pre>
<pre><code>[1] &quot;untreated&quot; &quot;crod&quot;      &quot;DSS&quot;      </code></pre>
<pre class="r"><code>DEGenesPdgfraloFb1 &lt;- FindAllMarkers(seuratPdgfraloFb1,only.pos=T) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01) %&gt;% mutate(celltype = &quot;PdgfraloFb1&quot;)
DEGenesPdgfraloFb1top &lt;- DEGenesPdgfraloFb1 %&gt;% top_n (200, avg_log2FC)

#top DE genes PdgfraloFb2
seuratPdgfraloFb2 &lt;- subset(seuratM, clusterName == &quot;PdgfraloFb2&quot;)
table(seuratPdgfraloFb2$clusterName)</code></pre>
<pre><code>
     PdgfraloFb1      PdgfraloFb2     Trophocytes1     Trophocytes2 Telocytes/Fibro2 
               0             4944                0                0                0 
      PdgfrahiFb         Myocytes            Glial             ICC1             ICC2 
               0                0                0                0                0 
          Immun1           Immun2              EpC          Cycling 
               0                0                0                0 </code></pre>
<pre class="r"><code>levels(seuratPdgfraloFb2)</code></pre>
<pre><code>[1] &quot;untreated&quot; &quot;crod&quot;      &quot;DSS&quot;      </code></pre>
<pre class="r"><code>DEGenesPdgfraloFb2 &lt;- FindAllMarkers(seuratPdgfraloFb2,only.pos=T) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01) %&gt;% mutate(celltype = &quot;PdgfraloFb2&quot;)
DEGenesPdgfraloFb2top &lt;- DEGenesPdgfraloFb2 %&gt;% top_n (200, avg_log2FC)

#top DE genes Trophocytes1
seuratTrophocytes1 &lt;- subset(seuratM, clusterName == &quot;Trophocytes1&quot;)
table(seuratTrophocytes1$clusterName)</code></pre>
<pre><code>
     PdgfraloFb1      PdgfraloFb2     Trophocytes1     Trophocytes2 Telocytes/Fibro2 
               0                0             5121                0                0 
      PdgfrahiFb         Myocytes            Glial             ICC1             ICC2 
               0                0                0                0                0 
          Immun1           Immun2              EpC          Cycling 
               0                0                0                0 </code></pre>
<pre class="r"><code>levels(seuratTrophocytes1)</code></pre>
<pre><code>[1] &quot;untreated&quot; &quot;crod&quot;      &quot;DSS&quot;      </code></pre>
<pre class="r"><code>DEGenesTrophocytes1 &lt;- FindAllMarkers(seuratTrophocytes1,only.pos=T) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01) %&gt;% mutate(celltype = &quot;Trophocytes1&quot;)
DEGenesTrophocytes1top &lt;- DEGenesTrophocytes1 %&gt;% top_n (200, avg_log2FC)

#top DE genes Trophocytes2
seuratTrophocytes2 &lt;- subset(seuratM, clusterName == &quot;Trophocytes2&quot;)
table(seuratTrophocytes2$clusterName)</code></pre>
<pre><code>
     PdgfraloFb1      PdgfraloFb2     Trophocytes1     Trophocytes2 Telocytes/Fibro2 
               0                0                0             1010                0 
      PdgfrahiFb         Myocytes            Glial             ICC1             ICC2 
               0                0                0                0                0 
          Immun1           Immun2              EpC          Cycling 
               0                0                0                0 </code></pre>
<pre class="r"><code>levels(seuratTrophocytes2)</code></pre>
<pre><code>[1] &quot;untreated&quot; &quot;crod&quot;      &quot;DSS&quot;      </code></pre>
<pre class="r"><code>DEGenesTrophocytes2 &lt;- FindAllMarkers(seuratTrophocytes2,only.pos=T) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01) %&gt;% mutate(celltype = &quot;Trophocytes2&quot;)
DEGenesTrophocytes2top &lt;- DEGenesTrophocytes2 %&gt;% top_n (200, avg_log2FC)

DEGenesAll &lt;- full_join(DEGenesPdgfraloFb1top, DEGenesPdgfraloFb2top)
DEGenesAll &lt;- full_join(DEGenesAll, DEGenesTrophocytes1top)
DEGenesAll &lt;- full_join(DEGenesAll, DEGenesTrophocytes2top)

#top DE genes Telocytes/Fibro2
seuratTelocytes &lt;- subset(seuratM, clusterName == &quot;Telocytes/Fibro2&quot;)
table(seuratTelocytes$clusterName)</code></pre>
<pre><code>
     PdgfraloFb1      PdgfraloFb2     Trophocytes1     Trophocytes2 Telocytes/Fibro2 
               0                0                0                0             5157 
      PdgfrahiFb         Myocytes            Glial             ICC1             ICC2 
               0                0                0                0                0 
          Immun1           Immun2              EpC          Cycling 
               0                0                0                0 </code></pre>
<pre class="r"><code>levels(seuratTelocytes)</code></pre>
<pre><code>[1] &quot;untreated&quot; &quot;crod&quot;      &quot;DSS&quot;      </code></pre>
<pre class="r"><code>DEGenesTelocytes &lt;- FindAllMarkers(seuratTelocytes,only.pos=T) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01) %&gt;% mutate(celltype = &quot;Telocytes/Fibro2&quot;)
DEGenesTelocytestop &lt;- DEGenesTelocytes %&gt;% top_n (200, avg_log2FC)

#top DE genes PdgfrahiFb
seuratPdgfrahiFb &lt;- subset(seuratM, clusterName == &quot;PdgfrahiFb&quot;)
table(seuratPdgfrahiFb$clusterName)</code></pre>
<pre><code>
     PdgfraloFb1      PdgfraloFb2     Trophocytes1     Trophocytes2 Telocytes/Fibro2 
               0                0                0                0                0 
      PdgfrahiFb         Myocytes            Glial             ICC1             ICC2 
            1436                0                0                0                0 
          Immun1           Immun2              EpC          Cycling 
               0                0                0                0 </code></pre>
<pre class="r"><code>levels(seuratPdgfrahiFb)</code></pre>
<pre><code>[1] &quot;untreated&quot; &quot;crod&quot;      &quot;DSS&quot;      </code></pre>
<pre class="r"><code>DEGenesPdgfrahiFb &lt;- FindAllMarkers(seuratPdgfrahiFb,only.pos=T) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01) %&gt;% mutate(celltype = &quot;PdgfrahiFb&quot;)
DEGenesPdgfrahiFbtop &lt;- DEGenesPdgfrahiFb %&gt;% top_n (200, avg_log2FC)

#top DE genes Myocytes
seuratMyocytes &lt;- subset(seuratM, clusterName == &quot;Myocytes&quot;)
table(seuratMyocytes$clusterName)</code></pre>
<pre><code>
     PdgfraloFb1      PdgfraloFb2     Trophocytes1     Trophocytes2 Telocytes/Fibro2 
               0                0                0                0                0 
      PdgfrahiFb         Myocytes            Glial             ICC1             ICC2 
               0             3162                0                0                0 
          Immun1           Immun2              EpC          Cycling 
               0                0                0                0 </code></pre>
<pre class="r"><code>levels(seuratMyocytes)</code></pre>
<pre><code>[1] &quot;untreated&quot; &quot;crod&quot;      &quot;DSS&quot;      </code></pre>
<pre class="r"><code>DEGenesMyocytes &lt;- FindAllMarkers(seuratMyocytes,only.pos=T) %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01) %&gt;% mutate(celltype = &quot;Myocytes&quot;)
DEGenesMyocytestop &lt;- DEGenesMyocytes %&gt;% top_n (200, avg_log2FC)

DEGenesAll &lt;- full_join(DEGenesPdgfraloFb1top, DEGenesPdgfraloFb2top)
DEGenesAll &lt;- full_join(DEGenesAll, DEGenesTrophocytes1top)
DEGenesAll &lt;- full_join(DEGenesAll, DEGenesTrophocytes2top)
DEGenesAll &lt;- full_join(DEGenesAll, DEGenesTelocytestop)
DEGenesAll &lt;- full_join(DEGenesAll, DEGenesPdgfrahiFbtop)
DEGenesAll &lt;- full_join(DEGenesAll, DEGenesMyocytestop)</code></pre>
</div>
<div id="distribution-of-log2fc-of-top-200-cw-de-genes"
class="section level2">
<h2>distribution of log2FC of top 200 cw DE genes</h2>
<pre class="r"><code>ggdensity(DEGenesAll, x = &quot;avg_log2FC&quot;, add= &quot;median&quot;, rug = TRUE, color = &quot;celltype&quot;, fill = &quot;celltype&quot;, palette = colclusterName)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/top%20100%20DE%20genes-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-top-100-DE-genes-1">
Past versions of "top 100 DE genes-1.png"
</button>
</p>
<div id="fig-no-spaces-top-100-DE-genes-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/38aa622eff0de8146d657049642a5b5dbe62aa67/docs/figure/MarkerGenes.Rmd/top 100 DE genes-1.png" target="_blank">38aa622</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggviolin(DEGenesAll, x = &quot;celltype&quot;, y = &quot;avg_log2FC&quot;, fill = &quot;celltype&quot;, palette = colclusterName, add = &quot;median_iqr&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/top%20100%20DE%20genes-2.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-top-100-DE-genes-2">
Past versions of "top 100 DE genes-2.png"
</button>
</p>
<div id="fig-no-spaces-top-100-DE-genes-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/38aa622eff0de8146d657049642a5b5dbe62aa67/docs/figure/MarkerGenes.Rmd/top 100 DE genes-2.png" target="_blank">38aa622</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggboxplot(DEGenesAll, x = &quot;celltype&quot;, y = &quot;avg_log2FC&quot;, color = &quot;celltype&quot;, palette = colclusterName ,add = &quot;jitter&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/top%20100%20DE%20genes-3.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-top-100-DE-genes-3">
Past versions of "top 100 DE genes-3.png"
</button>
</p>
<div id="fig-no-spaces-top-100-DE-genes-3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/38aa622eff0de8146d657049642a5b5dbe62aa67/docs/figure/MarkerGenes.Rmd/top 100 DE genes-3.png" target="_blank">38aa622</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dotplot-rig-i-signaling-pathway-genes" class="section level2">
<h2>dotplot RIG-I signaling pathway genes</h2>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$clusterName_cond

genes &lt;- data.frame(gene=rownames(seuratM)) %&gt;% 
  mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))

selGenes &lt;- data.frame(geneID=c(&quot;Ankrd17&quot;,&quot;Zc3hav1&quot;,&quot;Ddx60&quot;,&quot;Rigi&quot;,&quot;Pum2&quot;,&quot;Pum1&quot;,&quot;Nploc4&quot;,&quot;Usp15&quot;,&quot;Lsm14a&quot;,&quot;Nlrx1&quot;,&quot;Oasl1&quot;)) %&gt;% left_join(., genes, by=&quot;geneID&quot;)

DotPlot(seuratM, features = selGenes, group.by= &quot;clusterName_cond&quot;) + RotatedAxis() + scale_color_viridis(option=&quot;F&quot;) + coord_flip()</code></pre>
<p><img src="figure/MarkerGenes.Rmd/dotplot%20RIG-I-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-dotplot-RIG-I-1">
Past versions of "dotplot RIG-I-1.png"
</button>
</p>
<div id="fig-no-spaces-dotplot-RIG-I-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/af39455842ccf9a71a4fe05b21eee7e5e7d6d40a/docs/figure/MarkerGenes.Rmd/dotplot RIG-I-1.png" target="_blank">af39455</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="dotplot-activation-of-innate-immune-response"
class="section level2">
<h2>dotplot activation of innate immune response</h2>
<pre class="r"><code>Idents(seuratM) &lt;- seuratM$clusterName_cond

genes &lt;- data.frame(gene=rownames(seuratM)) %&gt;% 
  mutate(geneID=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))

egoDSS &lt;- dplyr::filter(egoDSS@result, egoDSS@result$Description ==&quot;activation of innate immune response&quot;)
g1 &lt;- egoDSS$geneID
Str &lt;- (g1)
strSub &lt;- strsplit(Str, &quot;/&quot;)
df &lt;- as.data.frame(strSub)
colnames(df) &lt;- c (&quot;geneID&quot;)

selGenes &lt;- data.frame(geneID=df$geneID) %&gt;% left_join(., genes, by=&quot;geneID&quot;)
DotPlot(seuratM, features = selGenes, group.by= &quot;clusterName_cond&quot;) + RotatedAxis() + scale_color_viridis(option=&quot;F&quot;) + coord_flip()</code></pre>
<p><img src="figure/MarkerGenes.Rmd/dotplot%20innate%20immune%20response-1.png" width="1152" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-dotplot-innate-immune-response-1">
Past versions of "dotplot innate immune response-1.png"
</button>
</p>
<div id="fig-no-spaces-dotplot-innate-immune-response-1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/angeldemartin/ColonicFb/blob/38aa622eff0de8146d657049642a5b5dbe62aa67/docs/figure/MarkerGenes.Rmd/dotplot innate immune response-1.png" target="_blank">38aa622</a>
</td>
<td>
angeldemartin
</td>
<td>
2024-12-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="subset-adamdec-pdgfralofb2-and-telocytesfibro2"
class="section level2">
<h2>subset Adamdec+ (PdgfraloFb2 and Telocytes/Fibro2)</h2>
<pre class="r"><code>seuratA &lt;- subset(seuratM, clusterName %in% c(&quot;PdgfraloFb2&quot;, &quot;Telocytes/Fibro2&quot;))
table(seuratA$clusterName)</code></pre>
<pre><code>
     PdgfraloFb1      PdgfraloFb2     Trophocytes1     Trophocytes2 Telocytes/Fibro2 
               0             4944                0                0             5157 
      PdgfrahiFb         Myocytes            Glial             ICC1             ICC2 
               0                0                0                0                0 
          Immun1           Immun2              EpC          Cycling 
               0                0                0                0 </code></pre>
</div>
<div id="calculate-de-genes-cond-1" class="section level2">
<h2>calculate DE genes cond</h2>
<pre class="r"><code>##DE genes DSS in Adamdec1+
Idents(seuratA) &lt;- seuratA$cond
levels(seuratA)
DEGenesAdamdec &lt;- FindAllMarkers(seuratA, only.pos=T)  %&gt;% 
  dplyr::filter(p_val_adj &lt; 0.01)

#save table
write.table(DEGenesAdamdec, 
            file= &quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/DEGenesAdamdec1_cond&quot;,
            sep=&quot;\t&quot;,
            quote=F,
            row.names=F,
            col.names=T)</code></pre>
</div>
<div id="gsea-dss-adamdec1" class="section level2">
<h2>GSEA DSS Adamdec1</h2>
<pre class="r"><code>## load DEGenesAdamdec1
DEGenesAdamdec &lt;- read.delim(&quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/DEGenesAdamdec1_cond&quot;, header = TRUE, sep = &quot;\t&quot;)

DEGenesAdamdec &lt;- DEGenesAdamdec %&gt;%
  mutate(Gene=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))  %&gt;%
  mutate(EnsID=gsub(&quot;\\..*&quot;,&quot;&quot;, gene))

DEDSSAdamdec1 &lt;- DEGenesAdamdec %&gt;% dplyr::filter(cluster == &quot;DSS&quot;)
##GSEA DSS
egoDSSAdamdec1&lt;- enrichGO(gene = unique(DEDSSAdamdec1$EnsID),
                          OrgDb = org.Mm.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egoDSSAdamdec1 &lt;- setReadable(egoDSSAdamdec1, OrgDb = org.Mm.eg.db)
dotplot(egoDSSAdamdec1, showCategory=30)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/GSEA%20DSS%20Adamdec1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="barplots-gsea-adamdec1" class="section level2">
<h2>barplots GSEA Adamdec1</h2>
<pre class="r"><code>#select pahtways immune response and filter file
selpathw &lt;- c(&quot;positive regulation of innate immune response&quot;, &quot;toll-like receptor 4 signaling pathway&quot;, &quot;regulation of NLRP3 inflammasome complex assembly&quot;, &quot;RIG-I signaling pathway&quot;, &quot;regulation of cytokine-mediated signaling pathway&quot;)

egoDSSAdamdec1_fil &lt;- egoDSSAdamdec1%&gt;% filter(egoDSSAdamdec1@result$Description %in% selpathw)

egoDSSAdamdec1_fil_2 &lt;- pairwise_termsim(egoDSSAdamdec1_fil)
#make ggbarplot
p &lt;- ggbarplot(egoDSSAdamdec1_fil_2@result, x = &quot;Description&quot;, y = &quot;Count&quot;, fill = &quot;#B45B5C&quot;, color= &quot;#B45B5C&quot;, sort.val = &quot;asc&quot;, orientation = &quot;horizontal&quot;)
p</code></pre>
<p><img src="figure/MarkerGenes.Rmd/barplots%20GSEA%20Adamdec1-1.png" width="1152" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#select pahtways epigenetic and filter file
selpathw &lt;- c(&quot;negative regulation of gene expression, epigenetic&quot;, &quot;epigenetic regulation of gene expression&quot;, &quot;positive regulation of gene expression, epigenetic&quot;)

egoDSSAdamdec1_fil &lt;- egoDSSAdamdec1%&gt;% filter(egoDSSAdamdec1@result$Description %in% selpathw)

egoDSSAdamdec1_fil_2 &lt;- pairwise_termsim(egoDSSAdamdec1_fil)
#make ggbarplot
p &lt;- ggbarplot(egoDSSAdamdec1_fil_2@result, x = &quot;Description&quot;, y = &quot;Count&quot;, fill = &quot;#B45B5C&quot;, color= &quot;#B45B5C&quot;, sort.val = &quot;asc&quot;, orientation = &quot;horizontal&quot;)
p</code></pre>
<p><img src="figure/MarkerGenes.Rmd/barplots%20GSEA%20Adamdec1-2.png" width="1152" style="display: block; margin: auto;" /></p>
</div>
<div id="violin-plots-adamdec1" class="section level2">
<h2>violin plots Adamdec1+</h2>
<pre class="r"><code>##positive regulation of gene expression, epigenetic
Idents(seuratA) &lt;- seuratA$cond
VlnPlot(object = seuratA, features = &quot;ENSMUSG00000086503.Xist&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000036940.Kdm1a&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-2.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000054203.Ifi205&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-3.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000024513.Mbd2&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-4.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000037487.Ubr5&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-5.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000030180.Kdm5a&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-6.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000026219.Trip12&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-7.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000078247.Airn&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-8.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000035851.Ythdc1&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-9.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000026313.Hdac4&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-10.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>##positive regulation of gene expression, epigenetic
VlnPlot(object = seuratA, features = &quot;ENSMUSG00000002028.Kmt2a&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-11.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000054611.Kdm2a&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-12.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000022536.Glyr1&quot;, pt.size= 1, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-13.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000031660.Brd7&quot;, pt.size= 1, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-14.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000055024.Ep300&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-15.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000055024.Ep300&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-16.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000025019.Lcor&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-17.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000038025.Phf2&quot;, pt.size= 0, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-18.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000085715.Tsix&quot;, pt.size= 1, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-19.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>VlnPlot(object = seuratA, features = &quot;ENSMUSG00000032187.Smarca4&quot;, pt.size= 1, cols= colcond) + theme(legend.position = &quot;none&quot;)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/violin%20plots%20Adamdec1+-20.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="gsea-crod-adamdec1" class="section level2">
<h2>GSEA crod Adamdec1</h2>
<pre class="r"><code>## load DEGenesAdamdec1
DEGenesAdamdec &lt;- read.delim(&quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/DEGenesAdamdec1_cond&quot;, header = TRUE, sep = &quot;\t&quot;)

DEGenesAdamdec &lt;- DEGenesAdamdec %&gt;%
  mutate(Gene=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))  %&gt;%
  mutate(EnsID=gsub(&quot;\\..*&quot;,&quot;&quot;, gene))

DEcrodAdamdec1 &lt;- DEGenesAdamdec %&gt;% dplyr::filter(cluster == &quot;crod&quot;)
##GSEA DSS
egocrodAdamdec1&lt;- enrichGO(gene = unique(DEcrodAdamdec1$EnsID),
                          OrgDb = org.Mm.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egocrodAdamdec1 &lt;- setReadable(egocrodAdamdec1, OrgDb = org.Mm.eg.db)
dotplot(egocrodAdamdec1, showCategory=30)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/GSEA%20crod%20Adamdec1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="gsea-untreated-adamdec1" class="section level2">
<h2>GSEA untreated Adamdec1</h2>
<pre class="r"><code>## load DEGenesAdamdec1
DEGenesAdamdec &lt;- read.delim(&quot;/Users/immbio/Desktop/Project/Angelina/ColonicFb/analysis/DEGenesAdamdec1_cond&quot;, header = TRUE, sep = &quot;\t&quot;)

DEGenesAdamdec &lt;- DEGenesAdamdec %&gt;%
  mutate(Gene=gsub(&quot;^.*\\.&quot;, &quot;&quot;, gene))  %&gt;%
  mutate(EnsID=gsub(&quot;\\..*&quot;,&quot;&quot;, gene))

DEuntreatedAdamdec1 &lt;- DEGenesAdamdec %&gt;% dplyr::filter(cluster == &quot;untreated&quot;)
##GSEA DSS
egountreatedAdamdec1&lt;- enrichGO(gene = unique(DEuntreatedAdamdec1$EnsID),
                          OrgDb = org.Mm.eg.db,
                          keyType = &#39;ENSEMBL&#39;,
                          ont = &quot;BP&quot;,
                          pAdjustMethod = &quot;BH&quot;,
                          pvalueCutoff = 0.05,
                          qvalueCutoff = 0.05)
egountreatedAdamdec1 &lt;- setReadable(egountreatedAdamdec1, OrgDb = org.Mm.eg.db)
dotplot(egountreatedAdamdec1, showCategory=30)</code></pre>
<p><img src="figure/MarkerGenes.Rmd/GSEA%20untreated%20Adamdec1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="session-info" class="section level2">
<h2>session info</h2>
<pre class="r"><code>date()</code></pre>
<pre><code>[1] &quot;Wed Mar  5 09:17:56 2025&quot;</code></pre>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-apple-darwin20
Running under: macOS Ventura 13.7.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] NCmisc_1.2.0                VennDiagram_1.7.3           futile.logger_1.4.3        
 [4] ggupset_0.4.1               gridExtra_2.3               DOSE_3.30.5                
 [7] enrichplot_1.24.4           msigdbr_7.5.1               org.Mm.eg.db_3.19.1        
[10] AnnotationDbi_1.66.0        clusterProfiler_4.12.6      multtest_2.60.0            
[13] metap_1.11                  scater_1.32.1               scuttle_1.14.0             
[16] destiny_3.18.0              circlize_0.4.16             muscat_1.18.0              
[19] viridis_0.6.5               viridisLite_0.4.2           lubridate_1.9.4            
[22] forcats_1.0.0               stringr_1.5.1               purrr_1.0.4                
[25] readr_2.1.5                 tidyr_1.3.1                 tibble_3.2.1               
[28] tidyverse_2.0.0             dplyr_1.1.4                 SingleCellExperiment_1.26.0
[31] SummarizedExperiment_1.34.0 Biobase_2.64.0              GenomicRanges_1.56.2       
[34] GenomeInfoDb_1.40.1         IRanges_2.38.1              S4Vectors_0.42.1           
[37] BiocGenerics_0.50.0         MatrixGenerics_1.16.0       matrixStats_1.5.0          
[40] pheatmap_1.0.12             ggpubr_0.6.0                ggplot2_3.5.1              
[43] Seurat_5.2.1                SeuratObject_5.0.2          sp_2.2-0                   
[46] runSeurat3_0.1.0            ExploreSCdataSeurat3_0.1.0 

loaded via a namespace (and not attached):
  [1] igraph_2.1.4              ica_1.0-3                 plotly_4.10.4            
  [4] Formula_1.2-5             zlibbioc_1.50.0           tidyselect_1.2.1         
  [7] bit_4.5.0.1               doParallel_1.0.17         clue_0.3-66              
 [10] lattice_0.22-6            rjson_0.2.23              blob_1.2.4               
 [13] S4Arrays_1.4.1            pbkrtest_0.5.3            parallel_4.4.0           
 [16] png_0.1-8                 plotrix_3.8-4             cli_3.6.4                
 [19] ggplotify_0.1.2           goftest_1.2-3             VIM_6.2.2                
 [22] variancePartition_1.34.0  BiocNeighbors_1.22.0      shadowtext_0.1.4         
 [25] uwot_0.2.2                curl_6.2.0                tidytree_0.4.6           
 [28] mime_0.12                 evaluate_1.0.3            ComplexHeatmap_2.20.0    
 [31] stringi_1.8.4             backports_1.5.0           lmerTest_3.1-3           
 [34] qqconf_1.3.2              httpuv_1.6.15             magrittr_2.0.3           
 [37] rappdirs_0.3.3            splines_4.4.0             ggraph_2.2.1             
 [40] sctransform_0.4.1         ggbeeswarm_0.7.2          DBI_1.2.3                
 [43] jquerylib_0.1.4           smoother_1.3              withr_3.0.2              
 [46] git2r_0.35.0              corpcor_1.6.10            reformulas_0.4.0         
 [49] class_7.3-23              rprojroot_2.0.4           lmtest_0.9-40            
 [52] tidygraph_1.3.1           formatR_1.14              colourpicker_1.3.0       
 [55] htmlwidgets_1.6.4         fs_1.6.5                  ggrepel_0.9.6            
 [58] labeling_0.4.3            fANCOVA_0.6-1             SparseArray_1.4.8        
 [61] DESeq2_1.44.0             ranger_0.17.0             DEoptimR_1.1-3-1         
 [64] reticulate_1.40.0         hexbin_1.28.5             zoo_1.8-12               
 [67] XVector_0.44.0            knitr_1.49                ggplot.multistats_1.0.1  
 [70] UCSC.utils_1.0.0          RhpcBLASctl_0.23-42       timechange_0.3.0         
 [73] foreach_1.5.2             patchwork_1.3.0           caTools_1.18.3           
 [76] ggtree_3.12.0             data.table_1.16.4         R.oo_1.27.0              
 [79] RSpectra_0.16-2           irlba_2.3.5.1             ggrastr_1.0.2            
 [82] gridGraphics_0.5-1        fastDummies_1.7.5         lazyeval_0.2.2           
 [85] yaml_2.3.10               survival_3.8-3            scattermore_1.2          
 [88] crayon_1.5.3              RcppAnnoy_0.0.22          RColorBrewer_1.1-3       
 [91] progressr_0.15.1          tweenr_2.0.3              later_1.4.1              
 [94] ggridges_0.5.6            codetools_0.2-20          GlobalOptions_0.1.2      
 [97] aod_1.3.3                 KEGGREST_1.44.1           Rtsne_0.17               
[100] shape_1.4.6.1             limma_3.60.6              pkgconfig_2.0.3          
[103] TMB_1.9.16                spatstat.univar_3.1-1     mathjaxr_1.6-0           
[106] EnvStats_3.0.0            aplot_0.2.4               scatterplot3d_0.3-44     
[109] ape_5.8-1                 spatstat.sparse_3.1-0     xtable_1.8-4             
[112] car_3.1-3                 plyr_1.8.9                httr_1.4.7               
[115] rbibutils_2.3             tools_4.4.0               globals_0.16.3           
[118] beeswarm_0.4.0            broom_1.0.7               nlme_3.1-167             
[121] lambda.r_1.2.4            lme4_1.1-36               digest_0.6.37            
[124] numDeriv_2016.8-1.1       Matrix_1.7-2              farver_2.1.2             
[127] tzdb_0.4.0                remaCor_0.0.18            reshape2_1.4.4           
[130] yulab.utils_0.2.0         glue_1.8.0                cachem_1.1.0             
[133] polyclip_1.10-7           generics_0.1.3            Biostrings_2.72.1        
[136] mvtnorm_1.3-3             presto_1.0.0              parallelly_1.42.0        
[139] mnormt_2.1.1              statmod_1.5.0             RcppHNSW_0.6.0           
[142] ScaledMatrix_1.12.0       carData_3.0-5             minqa_1.2.8              
[145] pbapply_1.7-2             httr2_1.1.0               spam_2.11-1              
[148] gson_0.1.0                graphlayouts_1.2.2        gtools_3.9.5             
[151] ggsignif_0.6.4            RcppEigen_0.3.4.0.2       shiny_1.10.0             
[154] GenomeInfoDbData_1.2.12   glmmTMB_1.1.10            R.utils_2.12.3           
[157] memoise_2.0.1             rmarkdown_2.29            scales_1.3.0             
[160] R.methodsS3_1.8.2         future_1.34.0             RANN_2.6.2               
[163] Cairo_1.6-2               spatstat.data_3.1-4       rstudioapi_0.17.1        
[166] cluster_2.1.8             whisker_0.4.1             mutoss_0.1-13            
[169] spatstat.utils_3.1-2      hms_1.1.3                 fitdistrplus_1.2-2       
[172] munsell_0.5.1             cowplot_1.1.3             colorspace_2.1-1         
[175] rlang_1.1.5               DelayedMatrixStats_1.26.0 sparseMatrixStats_1.16.0 
[178] xts_0.14.1                dotCall64_1.2             shinydashboard_0.7.2     
[181] ggforce_0.4.2             laeken_0.5.3              mgcv_1.9-1               
[184] xfun_0.50                 e1071_1.7-16              TH.data_1.1-3            
[187] iterators_1.0.14          abind_1.4-8               GOSemSim_2.30.2          
[190] treeio_1.28.0             futile.options_1.0.1      bitops_1.0-9             
[193] Rdpack_2.6.2              promises_1.3.2            scatterpie_0.2.4         
[196] RSQLite_2.3.9             qvalue_2.36.0             sandwich_3.1-1           
[199] fgsea_1.30.0              DelayedArray_0.30.1       proxy_0.4-27             
[202] GO.db_3.19.1              compiler_4.4.0            prettyunits_1.2.0        
[205] boot_1.3-31               beachmat_2.20.0           listenv_0.9.1            
[208] Rcpp_1.0.14               edgeR_4.2.2               workflowr_1.7.1          
[211] BiocSingular_1.20.0       tensor_1.5                MASS_7.3-64              
[214] progress_1.2.3            BiocParallel_1.38.0       babelgene_22.9           
[217] spatstat.random_3.3-2     R6_2.6.1                  fastmap_1.2.0            
[220] multcomp_1.4-28           fastmatch_1.1-6           rstatix_0.7.2            
[223] vipor_0.4.7               TTR_0.24.4                ROCR_1.0-11              
[226] TFisher_0.2.0             rsvd_1.0.5                vcd_1.4-13               
[229] nnet_7.3-20               gtable_0.3.6              KernSmooth_2.23-26       
[232] miniUI_0.1.1.1            deldir_2.0-4              htmltools_0.5.8.1        
[235] ggthemes_5.1.0            bit64_4.6.0-1             spatstat.explore_3.3-4   
[238] lifecycle_1.0.4           blme_1.0-6                nloptr_2.1.1             
[241] sass_0.4.9                vctrs_0.6.5               robustbase_0.99-4-1      
[244] spatstat.geom_3.3-5       sn_2.1.1                  ggfun_0.1.8              
[247] future.apply_1.11.3       bslib_0.9.0               pillar_1.10.1            
[250] gplots_3.2.0              pcaMethods_1.96.0         locfit_1.5-9.11          
[253] jsonlite_1.8.9            GetoptLong_1.0.5         </code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.4.0 (2024-04-24)
Platform: x86_64-apple-darwin20
Running under: macOS Ventura 13.7.1

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRblas.0.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.4-x86_64/Resources/lib/libRlapack.dylib;  LAPACK version 3.12.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Zurich
tzcode source: internal

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] NCmisc_1.2.0                VennDiagram_1.7.3           futile.logger_1.4.3        
 [4] ggupset_0.4.1               gridExtra_2.3               DOSE_3.30.5                
 [7] enrichplot_1.24.4           msigdbr_7.5.1               org.Mm.eg.db_3.19.1        
[10] AnnotationDbi_1.66.0        clusterProfiler_4.12.6      multtest_2.60.0            
[13] metap_1.11                  scater_1.32.1               scuttle_1.14.0             
[16] destiny_3.18.0              circlize_0.4.16             muscat_1.18.0              
[19] viridis_0.6.5               viridisLite_0.4.2           lubridate_1.9.4            
[22] forcats_1.0.0               stringr_1.5.1               purrr_1.0.4                
[25] readr_2.1.5                 tidyr_1.3.1                 tibble_3.2.1               
[28] tidyverse_2.0.0             dplyr_1.1.4                 SingleCellExperiment_1.26.0
[31] SummarizedExperiment_1.34.0 Biobase_2.64.0              GenomicRanges_1.56.2       
[34] GenomeInfoDb_1.40.1         IRanges_2.38.1              S4Vectors_0.42.1           
[37] BiocGenerics_0.50.0         MatrixGenerics_1.16.0       matrixStats_1.5.0          
[40] pheatmap_1.0.12             ggpubr_0.6.0                ggplot2_3.5.1              
[43] Seurat_5.2.1                SeuratObject_5.0.2          sp_2.2-0                   
[46] runSeurat3_0.1.0            ExploreSCdataSeurat3_0.1.0 

loaded via a namespace (and not attached):
  [1] igraph_2.1.4              ica_1.0-3                 plotly_4.10.4            
  [4] Formula_1.2-5             zlibbioc_1.50.0           tidyselect_1.2.1         
  [7] bit_4.5.0.1               doParallel_1.0.17         clue_0.3-66              
 [10] lattice_0.22-6            rjson_0.2.23              blob_1.2.4               
 [13] S4Arrays_1.4.1            pbkrtest_0.5.3            parallel_4.4.0           
 [16] png_0.1-8                 plotrix_3.8-4             cli_3.6.4                
 [19] ggplotify_0.1.2           goftest_1.2-3             VIM_6.2.2                
 [22] variancePartition_1.34.0  BiocNeighbors_1.22.0      shadowtext_0.1.4         
 [25] uwot_0.2.2                curl_6.2.0                tidytree_0.4.6           
 [28] mime_0.12                 evaluate_1.0.3            ComplexHeatmap_2.20.0    
 [31] stringi_1.8.4             backports_1.5.0           lmerTest_3.1-3           
 [34] qqconf_1.3.2              httpuv_1.6.15             magrittr_2.0.3           
 [37] rappdirs_0.3.3            splines_4.4.0             ggraph_2.2.1             
 [40] sctransform_0.4.1         ggbeeswarm_0.7.2          DBI_1.2.3                
 [43] jquerylib_0.1.4           smoother_1.3              withr_3.0.2              
 [46] git2r_0.35.0              corpcor_1.6.10            reformulas_0.4.0         
 [49] class_7.3-23              rprojroot_2.0.4           lmtest_0.9-40            
 [52] tidygraph_1.3.1           formatR_1.14              colourpicker_1.3.0       
 [55] htmlwidgets_1.6.4         fs_1.6.5                  ggrepel_0.9.6            
 [58] labeling_0.4.3            fANCOVA_0.6-1             SparseArray_1.4.8        
 [61] DESeq2_1.44.0             ranger_0.17.0             DEoptimR_1.1-3-1         
 [64] reticulate_1.40.0         hexbin_1.28.5             zoo_1.8-12               
 [67] XVector_0.44.0            knitr_1.49                ggplot.multistats_1.0.1  
 [70] UCSC.utils_1.0.0          RhpcBLASctl_0.23-42       timechange_0.3.0         
 [73] foreach_1.5.2             patchwork_1.3.0           caTools_1.18.3           
 [76] ggtree_3.12.0             data.table_1.16.4         R.oo_1.27.0              
 [79] RSpectra_0.16-2           irlba_2.3.5.1             ggrastr_1.0.2            
 [82] gridGraphics_0.5-1        fastDummies_1.7.5         lazyeval_0.2.2           
 [85] yaml_2.3.10               survival_3.8-3            scattermore_1.2          
 [88] crayon_1.5.3              RcppAnnoy_0.0.22          RColorBrewer_1.1-3       
 [91] progressr_0.15.1          tweenr_2.0.3              later_1.4.1              
 [94] ggridges_0.5.6            codetools_0.2-20          GlobalOptions_0.1.2      
 [97] aod_1.3.3                 KEGGREST_1.44.1           Rtsne_0.17               
[100] shape_1.4.6.1             limma_3.60.6              pkgconfig_2.0.3          
[103] TMB_1.9.16                spatstat.univar_3.1-1     mathjaxr_1.6-0           
[106] EnvStats_3.0.0            aplot_0.2.4               scatterplot3d_0.3-44     
[109] ape_5.8-1                 spatstat.sparse_3.1-0     xtable_1.8-4             
[112] car_3.1-3                 plyr_1.8.9                httr_1.4.7               
[115] rbibutils_2.3             tools_4.4.0               globals_0.16.3           
[118] beeswarm_0.4.0            broom_1.0.7               nlme_3.1-167             
[121] lambda.r_1.2.4            lme4_1.1-36               digest_0.6.37            
[124] numDeriv_2016.8-1.1       Matrix_1.7-2              farver_2.1.2             
[127] tzdb_0.4.0                remaCor_0.0.18            reshape2_1.4.4           
[130] yulab.utils_0.2.0         glue_1.8.0                cachem_1.1.0             
[133] polyclip_1.10-7           generics_0.1.3            Biostrings_2.72.1        
[136] mvtnorm_1.3-3             presto_1.0.0              parallelly_1.42.0        
[139] mnormt_2.1.1              statmod_1.5.0             RcppHNSW_0.6.0           
[142] ScaledMatrix_1.12.0       carData_3.0-5             minqa_1.2.8              
[145] pbapply_1.7-2             httr2_1.1.0               spam_2.11-1              
[148] gson_0.1.0                graphlayouts_1.2.2        gtools_3.9.5             
[151] ggsignif_0.6.4            RcppEigen_0.3.4.0.2       shiny_1.10.0             
[154] GenomeInfoDbData_1.2.12   glmmTMB_1.1.10            R.utils_2.12.3           
[157] memoise_2.0.1             rmarkdown_2.29            scales_1.3.0             
[160] R.methodsS3_1.8.2         future_1.34.0             RANN_2.6.2               
[163] Cairo_1.6-2               spatstat.data_3.1-4       rstudioapi_0.17.1        
[166] cluster_2.1.8             whisker_0.4.1             mutoss_0.1-13            
[169] spatstat.utils_3.1-2      hms_1.1.3                 fitdistrplus_1.2-2       
[172] munsell_0.5.1             cowplot_1.1.3             colorspace_2.1-1         
[175] rlang_1.1.5               DelayedMatrixStats_1.26.0 sparseMatrixStats_1.16.0 
[178] xts_0.14.1                dotCall64_1.2             shinydashboard_0.7.2     
[181] ggforce_0.4.2             laeken_0.5.3              mgcv_1.9-1               
[184] xfun_0.50                 e1071_1.7-16              TH.data_1.1-3            
[187] iterators_1.0.14          abind_1.4-8               GOSemSim_2.30.2          
[190] treeio_1.28.0             futile.options_1.0.1      bitops_1.0-9             
[193] Rdpack_2.6.2              promises_1.3.2            scatterpie_0.2.4         
[196] RSQLite_2.3.9             qvalue_2.36.0             sandwich_3.1-1           
[199] fgsea_1.30.0              DelayedArray_0.30.1       proxy_0.4-27             
[202] GO.db_3.19.1              compiler_4.4.0            prettyunits_1.2.0        
[205] boot_1.3-31               beachmat_2.20.0           listenv_0.9.1            
[208] Rcpp_1.0.14               edgeR_4.2.2               workflowr_1.7.1          
[211] BiocSingular_1.20.0       tensor_1.5                MASS_7.3-64              
[214] progress_1.2.3            BiocParallel_1.38.0       babelgene_22.9           
[217] spatstat.random_3.3-2     R6_2.6.1                  fastmap_1.2.0            
[220] multcomp_1.4-28           fastmatch_1.1-6           rstatix_0.7.2            
[223] vipor_0.4.7               TTR_0.24.4                ROCR_1.0-11              
[226] TFisher_0.2.0             rsvd_1.0.5                vcd_1.4-13               
[229] nnet_7.3-20               gtable_0.3.6              KernSmooth_2.23-26       
[232] miniUI_0.1.1.1            deldir_2.0-4              htmltools_0.5.8.1        
[235] ggthemes_5.1.0            bit64_4.6.0-1             spatstat.explore_3.3-4   
[238] lifecycle_1.0.4           blme_1.0-6                nloptr_2.1.1             
[241] sass_0.4.9                vctrs_0.6.5               robustbase_0.99-4-1      
[244] spatstat.geom_3.3-5       sn_2.1.1                  ggfun_0.1.8              
[247] future.apply_1.11.3       bslib_0.9.0               pillar_1.10.1            
[250] gplots_3.2.0              pcaMethods_1.96.0         locfit_1.5-9.11          
[253] jsonlite_1.8.9            GetoptLong_1.0.5         </code></pre>
</div>
</div>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
